modules/ROOT/pages/install-checklist.adoc

```diff
@@ -80,21 +81,26 @@ Don't configure SSL certificates in your load balancer. The platform handles the
 
 For more information, see the provider specific requirements:
 
+* https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html[Amazon EKS]
+* https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/bare-prereq/[Amazon EKS Anywhere (EKS-A)]
 * https://docs.rke2.io/install/requirements#hardware[Rancher Kubernetes Engine (RKE2) - Hardware]
 * https://docs.openshift.com/container-platform/4.16/installing/overview/index.html[Red Hat OpenShift]
-* https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/bare-prereq/[Amazon EKS Anywhere (EKS-A)]
+* https://techdocs.broadcom.com/us/en/vmware-cis/vcf/vsphere-supervisor-services-and-standalone-components/latest/managing-vsphere-kubernetes-service/running-tkg-service-clusters/tkg-service-components.html[vSphere Kubernetes Service (VKS)]
+
 
 == Software Requirements
 
 These are the requirements:
 
-* Operating system: RHEL 8.10, 9.4
+* Operating system: RHEL 8.10, 9.4, 9.6
 
 For more information, see the provider specific requirements:
 
+* https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html[Amazon EKS]
+* https://anywhere.eks.amazonaws.com/docs/getting-started/baremetal/bare-prereq/[Amazon EKS Anywhere (EKS-A)]
 * https://docs.rke2.io/install/requirements#operating-systems[Rancher Kubernetes Engine (RKE2) - Software]
 * https://docs.openshift.com/container-platform/4.16/installing/overview/index.html[Red Hat OpenShift]
```

Comment:
Please remove the reference to any Openshift documentation for PCE 4.1.2 release